SalesforceCommerceCloud/pwa-kit
# Allow query hook parameters to be `null`.

When the app is initializing, some values (e.g. `customerId`) can be unset. React doesn't allow conditionally calling hooks. Therefore, we must support an explicit "unset" value for parameters. As it is not otherwise used by the Commerce API or isomorphic SDK, `null` is a natural choice for the fallback value.



# Description



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Use a hook. Pass `null` as a value. Observe that TypeScript does not complain, and the hook is not executed.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
